### PR TITLE
Adds a 'Contact Us' link to the Self-Hosted pages.

### PR DESCRIPTION
**Pull request by clstokes**
Fixes https://github.com/pulumi/docs/issues/5202.

Used `<a href>...</a>` in all places to be consistent with the `div` at the top of the page, where `relref` doesn't seem to work.

### CODE REVIEW

File: content/docs/guides/self-hosted/_index.md [hunk 1]
```diff
@@ -51,7 +49,7 @@ Here are some examples of deployment topologies:
 | [Console]({{< relref "console" >}}) |	[https://hub.docker.com/r/pulumi/console/](https://hub.docker.com/r/pulumi/console/) |
 | Migrations | [https://hub.docker.com/r/pulumi/migrations/](https://hub.docker.com/r/pulumi/migrations/) |
 
-> **Note**: The above container image repositories are private.
+> **Note**: The above container image repositories are private. [Contact us]({{< relref "/contact.md" >}}) if you would like to evaluate the Self-Hosted Enterprise Edition.
```
Comment (davidwrede) [inline content/docs/guides/self-hosted/_index.md:52]:
```suggestion
{{% notes type="info" %}}
The above container image repositories are private. [Contact us]({{< relref "/contact.md" >}}) if you would like to evaluate the Self-Hosted Enterprise Edition.
{{% /notes %}}
```

File: content/docs/guides/self-hosted/api.md [hunk 2]
```diff
@@ -38,6 +36,8 @@ The Pulumi API is one of the components required for self-hosting Pulumi in your
 
 ## What's In The Container?
 
+> **Note**: The container image repository is private. [Contact us]({{< relref "/contact.md" >}}) if you would like to evaluate the Self-Hosted Enterprise Edition.
```
Comment (davidwrede) [inline content/docs/guides/self-hosted/api.md:39]:
```suggestion
{{% notes type="info" %}}
The container image repository is private. [Contact us]({{< relref "/contact.md" >}}) if you would like to evaluate the Self-Hosted Enterprise Edition.
{{% /notes %}}
```

File: content/docs/guides/self-hosted/console.md [hunk 3]
```diff
@@ -36,6 +34,8 @@ You can run this container on the same host that your API container is running o
 
 ## What's In The Container?
 
+> **Note**: The container image repository is private. [Contact us]({{< relref "/contact.md" >}}) if you would like to evaluate the Self-Hosted Enterprise Edition.
```
Comment (davidwrede) [inline content/docs/guides/self-hosted/console.md:37]:
```suggestion
{{% notes type="info" %}}
The container image repository is private. [Contact us]({{< relref "/contact.md" >}}) if you would like to evaluate the Self-Hosted Enterprise Edition.
{{% /notes %}}
```

**Review by davidwrede — COMMENTED** [review]
Left suggestions on how to standardize the remaining notes in this topic.

**Review by infin8x — APPROVED** [review]
LGTM

**Review by davidwrede — APPROVED** [review]

**Pull request merged**